.travis.yml

```diff
@@ -142,12 +126,33 @@ matrix:
             - libboost-log1.55-dev
             - ninja-build
 
+
 before_script:
-  - ./install_dependencies.sh
-  - export PATH=$PATH:$PWD/odb/odb-2.4.0-x86_64-linux-gnu/bin/
-  - cmake -DBUILD_STATIC=1 -DCMAKE_CXX_COMPILER=${CXX_COMPILER} -DCMAKE_C_COMPILER=${C_COMPILER} -DCMAKE_CXX_FLAGS="-Iodb/libodb-2.4.0 -Iodb/libodb-sqlite-2.4.0" -DCMAKE_EXE_LINKER_FLAGS="-Wl,-rpath,odb/ -Lodb/"  -DODB_PATH=$PWD/odb/ -G Ninja
+  - |
+    if [[ $TRAVIS_OS_NAME == "linux" ]]; then
+      ./install_dependencies.sh
+      export PATH=$PATH:$PWD/linux_dependencies/odb-2.4.0-x86_64-linux-gnu/bin/
+    fi
+  - |
+    if [[ $TRAVIS_OS_NAME == "osx" ]]; then
+      ./install_dependencies.sh osx
+      export PATH=$PATH:$PWD/osx_dependencies/odb-2.4.0-i686-macosx/bin/
+    fi
+
 
 script:
-  - ninja -j 1 -v
-  - ninja test
+  - |
+    if [[ $TRAVIS_OS_NAME == "linux" ]]; then
+      cmake -DCMAKE_CXX_COMPILER=${CXX_COMPILER} -DCMAKE_C_COMPILER=${C_COMPILER} -DCMAKE_CXX_FLAGS="-Ilinux_dependencies/libodb-2.4.0 -Ilinux_dependencies/libodb-sqlite-2.4.0" -DCMAKE_EXE_LINKER_FLAGS="-Wl,-rpath,linux_dependencies/ -Llinux_dependencies/"  -DEXT_LIB_PATH=$PWD/linux_dependencies/ ./ -GNinja
+    fi
+  - |
+    if [[ $TRAVIS_OS_NAME == "osx" ]]; then
+      cmake -DCMAKE_CXX_COMPILER=${CXX_COMPILER} -DCMAKE_C_COMPILER=${C_COMPILER} -DCMAKE_CXX_FLAGS="-Iosx_dependencies/libodb-2.4.0 -Iosx_dependencies/libodb-sqlite-2.4.0" -DCMAKE_EXE_LINKER_FLAGS="-Wl,-rpath,osx_dependencies/ -Losx_dependencies/"  -DEXT_LIB_PATH=$PWD/osx_dependencies/ ./ -GNinja
+    fi
 
+  - ninja -v -j2
+  - ninja test
+  - |
+    if [[ $TRAVIS_OS_NAME == "osx" ]]; then
+      otool -L bin/vcf_validator
```

Comment:
I don't think we need this now that we know the static build works, but I'm not strongly against it either.

Reply:
I think we can keep this. So that we can easlily keep track of the dynamic dependencies added in future.